src/commands/goto.ts

```diff
@@ -50,6 +59,70 @@ function executeGoto(gotoType: number, editor: vscode.TextEditor, count: number,
   }
 }
 
+function executeGotoLine(editor: vscode.TextEditor, count: number, extend: boolean, position: string = 'default') {
```

Comment:
`position` should have more precise type (`'first' | 'end' | 'start' | 'default'`).